docs/prometheus-metrics.md

```diff
@@ -109,3 +113,7 @@ The configuration file above can be used to deploy Prometheus Server with
 scraping configuration for Antrea services.
 To deploy this configuration use
 `kubectl apply -f build/yamls/antrea-prometheus.yml`
+
+# Deprecation Policy
+
+In Antrea, Prometheus metric deprecation policy is to flag them deprecated in `Antrea 0.N.0` by adding `(Deprecated since Antrea 0.N.0)` in the front of metric's help text. The release target of removing the deprecated metrics would be `Antrea 0.N+1.0`. We follow the policy given in the [Kubernetes Metrics Overhaul](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/20181106-kubernetes-metrics-overhaul.md#deprecation-plan) document.
```

Comment:
let's not make this part of this PR, I don't want it to go unnoticed

Reply:
Removed this.